lib/pp.rb

```diff
@@ -223,7 +223,16 @@ def seplist(list, sep=nil, iter_method=:each) # :yield: element
         else
           sep.call
         end
-        yield(*v)
+        case v.last
+        when Hash
+          if Hash.ruby2_keywords_hash?(v.last)
```

Comment:
Doesn't `Hash.ruby2_keywords_hash?` allow any object?
If so it could be simplified to:
```ruby
if Hash.ruby2_keywords_hash?(v.last)
  yield(*v, **{})
else
  yield(*v)
end
```

Reply:
No, I tried that first.

Reply:
Mmh, that's really unfortunate, I'll file a feature request for it.

Reply:
https://bugs.ruby-lang.org/issues/16697